wpengine/faustjs
# MERL-803: overriding introspection

## Tasks

## Description
Adds support for authenticated introspection queries, so that users are not required to turn on "Enable Public Introspection" in WPGraphQL.

## Testing
- Set up a WP site with the Faust plugin, WPGraphQL, etc.
- Make sure WPGraphQL does **not** have "Enable Public Introspection" turned on.
- Set up a Faust example project, configured to work with the WP site above.
- Check out `canary` branch
- Run `npm run generate` in your Faust app. See error about needing to enable public introspection.
- Check out this PR branch. Ensure both your WP and Faust app are using this branch.
- Build the Faust app
- Run `npm run generate`. See success.
- Try it with an incorrect secret key to see updated error messaging.
## Screenshots



## Documentation Changes
